Sept-2021-Cohort/Sept-2021-02-react.md

```diff
@@ -1,195 +1,139 @@
 
-* [Deployed Site]()
+---
 
->| Team Members  | Github  | LinkedIn  |
->|---|---|---|
->| Name| [Github]()   | [LinkedIn]()   |
->|  Name| [Github]()  |  [LinkedIn]()  |
->|  Name| [Github]()  |  [LinkedIn]()  |
->| Name |  [Github]()  |  [LinkedIn]()  |
+---
 
-___
-___
+### Yahfood
 
-### 
+> Description
+>
+> > Yah-Food is a fun, simple, and creative app that helps the user discover new foods based on the ingredients they already have. It offers a small featured recipe section, ability to favorite certain recipes, and create your own recipes. This App is suited for both mobile and desktop use.
 
-> Description 
->>
+- [Repo](https://github.com/Yah-food/Yahfood-final)
 
-* [Repo]()
+- [Deployed Site](https://hopeful-curie-14d366.netlify.app/)
 
-* [Deployed Site]()
+> | Team Members     | Github                                        | LinkedIn                                                 |
+> | ---------------- | --------------------------------------------- | -------------------------------------------------------- |
+> | Andrew Joy       | [Github](https://github.com/ajoy267)                                    | [LinkedIn](https://www.linkedin.com/in/andrewjoy12/)                                             |
+> | Kevin Do         | [Github](https://github.com/kevindo1)         | [LinkedIn](https://www.linkedin.com/in/kdo/)             |
+> | Minoka Kakizaki  | [Github](https://github.com/kakizaki55)       | [LinkedIn](https://www.linkedin.com/in/minoka-kakizaki/) |
+> | Spencer Eagleton | [Github](https://github.com/spencer-eagleton) | [LinkedIn](https://www.linkedin.com/in/spencer-eagleton/)                                             |
 
->| Team Members  | Github  | LinkedIn  |
->|---|---|---|
->| Name| [Github]()   | [LinkedIn]()   |
->|  Name| [Github]()  |  [LinkedIn]()  |
->|  Name| [Github]()  |  [LinkedIn]()  |
->| Name |  [Github]()  |  [LinkedIn]()  |
+---
 
-___
-___ -->
-<!-- ### 
+---
 
-> Description 
->>
+### TimeHub
 
-* [Repo]()
+> Description
+>
+> > An app that visually records your emotions over time with a colormap through journal entries.
 
-* [Deployed Site]()
+- [Repo](https://github.com/Time-Hub-App/Time-Hub-App)
 
->| Team Members  | Github  | LinkedIn  |
->|---|---|---|
->| Name| [Github]()   | [LinkedIn]()   |
->|  Name| [Github]()  |  [LinkedIn]()  |
->|  Name| [Github]()  |  [LinkedIn]()  |
->| Name |  [Github]()  |  [LinkedIn]()  |
+- [Deployed Site](https://time-hub-2.netlify.app/)
 
-___
-___
+> | Team Members | Github     | LinkedIn     |
+> | ------------ | ---------- | ------------ |
+> |  Ckay Natividad         | [Github](https://github.com/ckaynatividad) | [LinkedIn](https://www.linkedin.com/in/ckaynatividad) |
+> | Libbi Dunham         | [Github](https://github.com/Libbi-Dunham) | [LinkedIn](https://www.linkedin.com/in/libbi-dunham/) |
+> | Abriel Cleaver         | [Github]() | [LinkedIn]() |
```

Comment:
We just took out the links for people who didn't give us their info. That way it's more clear who hasn't provided their info